web/virtual_lab/static/virtual_lab/js/code_editor.js

```
@@ -0,0 +1,69 @@
+// CSRF helper
+function getCookie(name) {
+  let cookieValue = null;
+  document.cookie.split(';').forEach(c => {
+    c = c.trim();
+    if (c.startsWith(name + '=')) {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)